docs/support_playbook.md

```diff
@@ -195,7 +195,7 @@ trainee.update(state: :trn_received, recommended_for_award_at: nil, awarded_at:
 
 ```
 
-All record changes should be sent to DQT unless otherwise specified or impossible (e.g. we can't send a DOB update). If DQT already has that info (e.g. they're awarded on DQT, and we're just awarding on Register) we should not send any information.
+All record changes should be sent to DQT unless otherwise specified or impossible (e.g. we can’t send a DOB update). If DQT already has that info (e.g. they’re awarded on DQT, and we’re just awarding on Register) we should not send any information.
```

Comment:
```suggestion
All record changes should be sent to DQT unless otherwise specified or impossible (for example, we cannot send a DOB update). If DQT already has that info (for example, they’re awarded on DQT, and we’re just awarding on Register) we should not send any information.
```